src/EditorFeatures/Test/CodeLens/AbstractCodeLensTest.cs

```diff
@@ -16,6 +16,9 @@ public abstract class AbstractCodeLensTest
         {
             using (var workspace = await TestWorkspace.CreateAsync(input))
             {
+                var service = workspace.GetService<ICodeLensReferencesService>();
```

Comment:
Thanks, this was me thinking about how to reuse this code for test, but I'll stick with the original way.